### DISCUSSION TITLE
Как исправить округление при умножении на значение из селекта?

### DISCUSSION THREAD

**Discussion by Skjorwald** [totumonline/totum-ru-issues-and-discussions#171] [Как это сделать?] [answered]
Добрый день.

Столкнулся с такой проблемой: создал формулу умножения, одним из множителей которого является значение, выбранное из селекта. При умножении на значение, выбранное из селекта, производится округление результата до целого числа. Возможно ли (и как) убрать это округление?
<img width="1136" alt="Снимок экрана 2022-07-25 в 14 15 36" src="https://user-images.githubusercontent.com/109959960/180765474-e2ae4f36-aff4-4a06-b9dd-22b753e3866f.png">

**Comment by erpic-ru** [comment, marked as answer]
Потому что он умножает не на отображение, а на основание селекта. В данном случае - это id со значением 3. Чтобы умножалось на то, что видно 
в кодах следует использовать`#s.koefficeient_obmena`. Т. е поставить "s." после решетки. 

https://ru.docs.totum.online/syntax#part-6

**Reply by Skjorwald** [reply]
Спасибо, все получилось. Запомню на будущее.
